upload/install/cli_install.php

```diff
@@ -123,6 +123,13 @@ public function index(): void {
 		$this->response->setOutput($output);
 	}
 
+	/**
+	 * Install
+	 * 
+	 * @param $argv
```

Comment:
This is invalid without a type

Reply:
Ok, closing commit. Will re-open a new one.

Reply:
You can also update a branch by adding more commits to it to adjust and update the pr, but if you feel it's easier to make a new one that is also OK

Reply:
I created a new PR: https://github.com/opencart/opencart/pull/13293 . All passed PHP versions.